### DISCUSSION TITLE
Screen resolution issues

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5496] [Q&A] [answered]
I recently installed an i3 and ran into a problem, on my 4k monitor the whole image is very small. I tried setting it to 1920x1080, but then my screen got all pixelated. I assume it might be related to dpi (I tried changing the values via xrandr, but nothing worked).

**Comment by orestisfl** [comment, marked as answer]
What did you try exactly? 

There are some instructions on https://i3wm.org/docs/userguide.html#hidpi and the linked arch Linux guide
